packages/create-gasket-app/test/unit/scaffold/actions/write-gasket-config.test.js

```diff
@@ -34,58 +32,85 @@ describe('write-gasket-config', () => {
+    await writeGasketConfig(null, mockContext);
+    const output = mockWriteStub.mock.calls[0][1];
+    expect(output).toContain('export default');
   });
 
-  it('writes gasket.config.js with module.exports', async () => {
-    mockWriteStub.mockResolvedValue();
-    await writeGasketConfig(mockContext);
+  it('writes @gasket/core import', async () => {
+    await writeGasketConfig(null, mockContext);
     const output = mockWriteStub.mock.calls[0][1];
-    expect(output).toContain('module.exports = ');
+    expect(output).toContain('import { makeGasket } from \'@gasket/core\';');
   });
 
-  it('writes pretty JSON5 from gasketConfig', async () => {
-    mockWriteStub.mockResolvedValue();
-    await writeGasketConfig(mockContext);
+  it('writes gasket.js with makeGasket', async () => {
+    await writeGasketConfig(null, mockContext);
     const output = mockWriteStub.mock.calls[0][1];
-    const expected = JSON5.stringify(mockContext.gasketConfig, null, 2);
-    expect(output).toContain(expected + ';\n');
+    expect(output).toContain('makeGasket(');
   });
 
   it('does not double-quote keys', async () => {
-    mockWriteStub.mockResolvedValue();
-    await writeGasketConfig(mockContext);
+    await writeGasketConfig(null, mockContext);
     const output = mockWriteStub.mock.calls[0][1];
-    expect(output).toContain('plugins: {');
+    expect(output).toContain('plugins:');
   });
 
-  it('Adds preset to config', async () => {
-    mockContext.presets = ['bogus-preset'];
-    mockWriteStub.mockResolvedValue();
-    await writeGasketConfig(mockContext);
+  it('outputs keys without quotes, strings with single-quotes', async () => {
+    mockContext.gasketConfig.add("bogus", "double"); // eslint-disable-line quotes
+    await writeGasketConfig(null, mockContext);
     const output = mockWriteStub.mock.calls[0][1];
-    expect(output).toContain('presets: [');
-    expect(output).toContain('\'bogus-preset\'');
+    expect(output).toContain('bogus: \'double\'');
   });
 
-  it('Adds extra plugins to config', async () => {
-    mockContext.plugins = ['bogus-plugin', 'another-plugin'];
-    mockWriteStub.mockResolvedValue();
-    await writeGasketConfig(mockContext);
+  it('writes plugin imports', async () => {
+    mockContext.gasketConfig.fields.plugins = ['pluginBogus', 'pluginBogus2'];
+    mockContext.gasketConfig.fields.pluginImports = {
+      pluginBogus: '@gasket/plugin-bogus',
+      pluginBogus2: '@gasket/plugin-bogus2'
+    };
+    await writeGasketConfig(null, mockContext);
     const output = mockWriteStub.mock.calls[0][1];
-    expect(output).toContain('add: [');
-    expect(output).toContain('\'bogus-plugin\'');
-    expect(output).toContain('\'another-plugin\'');
+    expect(output).toContain('import pluginBogus from \'@gasket/plugin-bogus\';');
+    expect(output).toContain('import pluginBogus2 from \'@gasket/plugin-bogus2\';');
+    expect(output.match(/\[(\s.*)+\]/)[0]).toBe('[\n\t\tpluginBogus,\n\t\tpluginBogus2\n\t]');
```

Comment:
nit: prefer to use the [Jest matchers](https://jestjs.io/docs/expect) instead of inspecting results of a RegExp. It's a little easier to understand and update.

```suggestion
    expect(output).toEqual(expect.stringMatching(/plugins: \[\s+pluginBogus,\s+pluginBogus2\s+]/));
```